xml/System.Text.Json/JsonDocumentOptions.xml

```diff
@@ -81,7 +81,7 @@ By default, `AllowTrailingCommas` is set to `false`, and a <xref:System.Text.Jso
 
 ## Remarks
 
-By default, a <exception cref="T:System.Text.Json.JsonException> is thrown if a comment is encountered.
+By default, a <xref:System.Text.Json.JsonException> is thrown if a comment is encountered.
```

Comment:
Can't believe we didn't catch this before. The build should've failed.

Reply:
Yes. It's disturbing that it didn't.